src/modules/transaction/hooks/useTransactionFee/utils.js

```diff
@@ -0,0 +1,63 @@
+import { transactions } from '@liskhq/lisk-client';
+import { joinModuleAndCommand } from '@transaction/utils/moduleCommand';
+import { DEFAULT_SIGNATURE_BYTE_SIZE } from '@transaction/configuration/transactions';
+
+export const ZERO_FEE = BigInt(0);
+
+/**
+ * This is solution uses the same logic as implemented in transactions utility.
+ * We'll replace this easily with said class methods.
+ *
+ * @param {object} transaction transaction object in JSON format
+ * @param {object} paramsSchema transaction schema as retrieved from Service
+ * @param {object} auth the auth WS response data
+ * @param {array} priorities list of objects with value (number) and title (string), and selected (boolean)
+ * @param {boolean} isTxValid defines if the form was validated successfully
+ * @param {bigint} extraFee
+ *
+ * @returns {bigint} the transaction fee in Beddows
+ */
+export const computeTransactionMinFee = (
+  transaction,
+  paramsSchema,
+  auth,
+  priorities,
+  isTxValid,
+) => {
+  if (!isTxValid || !paramsSchema || !auth || !priorities?.length) return ZERO_FEE;
+
+  const allocateEmptySignaturesWithEmptyBuffer = (signatureCount) =>
+    new Array(signatureCount).fill(Buffer.alloc(DEFAULT_SIGNATURE_BYTE_SIZE));
```

Comment:
We can eliminate this logic by passing the `options`? https://github.com/LiskHQ/lisk-sdk/blob/723817d2998270a6ea06d545d72adfca46a9d129/elements/lisk-transactions/src/fee.ts#L62